Assets/Scripts/UI/SettingsMenu.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections.Generic;
+using FrameWork;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Audio;
+using Slider = UnityEngine.UI.Slider;
+
+namespace UI
+{
+    public class SettingsMenu : MonoBehaviour
+    {
+        [SerializeField] private AudioMixer audioMixer;
+        [SerializeField] private TMP_Dropdown graphicsDropdown;
+        [SerializeField] private TMP_Dropdown resolutionsDropdown;
+        [SerializeField] private Slider slider;
+        [SerializeField] private Slider progressSlider;
+        [SerializeField] private bool customSettings;
+        [SerializeField] private SceneSwitcher _sceneSwitcher;
```

Comment:
Dit zijn heel wat varibles die je ziet in de inspector, kan je wat headers toevoegen.

+ _sceneSwitcher moet zonder `_`

```suggestion
        [SerializeField] private AudioMixer audioMixer;
        [SerializeField] private TMP_Dropdown graphicsDropdown;
        [SerializeField] private TMP_Dropdown resolutionsDropdown;
        [SerializeField] private Slider slider;
        [SerializeField] private Slider progressSlider;
        [SerializeField] private bool customSettings;
        [SerializeField] private SceneSwitcher sceneSwitcher;
```